lib/core/versions/latest/InputValidator.ts

```diff
@@ -0,0 +1,37 @@
+import ErrorCode from './ErrorCode';
+import SidetreeError from '../../../common/SidetreeError';
+
+/**
+ * Class containing generic input validation methods.
+ */
+export default class InputValidator {
```

Comment:
Should this be versioned? I feel like this should be a general purposed helper. Or at the very least be in util?

Reply:
I think it is fine to be versioned (Based on the guiding principle of: put as much code into versioned folder as possible).
Good ask about putting the code in `util`, I also looked at the `util` folder but decided against it because the classes in there have nothing to do with Sidetree, and this one is arguable.